bundles/org.smarthomej.binding.amazonechocontrol/src/main/java/org/smarthomej/binding/amazonechocontrol/internal/AccountServlet.java

```diff
@@ -396,8 +390,9 @@ private void handleDefaultPageResult(HttpServletResponse resp, String message, C
         html.append("/changeDomain'>Change</a>");
 
         // Main UI link
-        html.append("<br><a href='/#!/settings/things/" + BINDING_ID + ":"
-                + URLEncoder.encode(THING_TYPE_ACCOUNT.getId(), "UTF8") + ":" + URLEncoder.encode(id, "UTF8") + "'>");
+        html.append("<br><a href='/#!/settings/things/" + BINDING_ID + ":")
```

Comment:
There are two more "+" signs here. Did you leave them in place by intention?

Reply:
No. Just didn't see them because they were not underlined as error in IntelliJ. Thanks.